pkg/controller/windowsmachineconfig/windows/windows.go

```diff
@@ -259,6 +265,96 @@ func (vm *Windows) ConfigureCNI(configFile string) error {
 	return nil
 }
 
+// service represents a Windows service
+type service struct {
+	// binPath includes a path to the binary and all arguments to run the binary with
+	binPath string
+	// name is the name of the service
+	name string
+	// vm is the VM the service will be started on
+	vm *Windows
```

Comment:
This seems backward to me. A VM struct should have a service object or a list of service objects and not the other way around. It is more natural to have `vm.service` rather than `service.vm`. This is what @ravisantoshgudimetla is [calling out too](https://github.com/openshift/windows-machine-config-operator/pull/65/commits/55d23940ec6fbb6a0dbd97c11ff3c95cc36f462d#r436996588). So in addition to `binPath`, `name` etc you will have `createCmd`, `startCmd` that `vm` will act upon.